modules/p2/ding_entity_rating/ding_entity_rating.module

```diff
@@ -38,26 +38,72 @@ function ding_entity_rating_menu() {
     'delivery callback' => 'ajax_deliver',
   );
 
-  $items['ding_entity_rating/get'] = array(
-    'title' => 'Ding entity get rating',
-    'page callback' => 'ding_entity_rating_get_user_rating_callback',
+  $items['ding_entity_rating/user/get'] = array(
```

Comment:
All these changes to the rating module are not relevant to the change at hand. Has something gone wrong with a merge or a rebase?